sim/core/consumes.go

```diff
@@ -514,6 +517,9 @@ func registerPotionCD(agent Agent, consumes *proto.Consumes) {
 				if defaultPotion == proto.Potions_RunicManaPotion {
 					character.ExpectedBonusMana -= float64((4200 + 4400) / 2)
 				}
+				if defaultPotion == proto.Potions_RunicManaInjector {
+					character.ExpectedBonusMana -= (4200 + 4400) / 2 * 1.25
```

Comment:
do you need the isEngi conditional on the x1.25 there?

Reply:
yep, good catch, fixed